blog/_posts/2022-02-04-policy-objects-routing-example.md

```diff
@@ -31,13 +31,9 @@ After much research and troubleshooting I learned that firewalld was rejecting i
 
 ## How do we fix it?
 
-This tells me the FedoraServer zone is rejecting my request.  Why?  It turns out that the Firewalld+nftables filter_FORWARD chain prevents non-NATed routing.  The feature implementation to support is a work in progress at the time of writing this article, and more info can be found here:
+This tells me that the FedoraServer zone is rejecting my request.  Why?  It turns out that the Firewalld+nftables filter_FORWARD chain prevents non-NATed routing, and firewalld will deny by default.  However, forwarding is possible using the "Policy Objects" feature that was implemented in firewalld around 2020.
```

Comment:
Specifically, policy objects were added in v0.9.0.

Reply:
> It turns out that the Firewalld+nftables filter_FORWARD chain prevents non-NATed routing

NATed connections will also be denied by default. The forward filtering occurs _before_ the NAT operations. So the forwarded traffic needs to be allowed in both cases: NAT and non-NAT.